### DISCUSSION TITLE
Pipes and pumps

### DISCUSSION THREAD

**Discussion by Seegras** [TechReborn/TechReborn#2982] [Ideas]
Given that Tech Reborn does such a lot of things with fluids, and some machines even require it, it should also have at least its own fluid pipes; and maybe a pump.

**Comment by Ayutac** [comment]
You can use tanks as pumps and pipes

**Reply by axelrindle** [reply]
Tanks are not very practical for transporting fluids or items over greater distances.

**Comment by KhrystynaKurylas** [comment]
This is a good idea. Needs item sorting pipes and fluids pipes

**Reply by Technici4n** [reply]
https://www.curseforge.com/minecraft/mc-mods/modern-dynamics

**Reply by KhrystynaKurylas** [reply]
This other mod. Oh 1.18.2(( no 1.19

**Reply by Technici4n** [reply]
I'll port it to 1.19 soon.
